src/main/java/org/kairosdb/datastore/cassandra/cache/AbstractByteBufferCache.java

```diff
@@ -41,4 +49,18 @@ BigInteger doubleHash(final ByteBuffer payload) {
         doubleHash.flip();
         return new BigInteger(doubleHash.array());
     }
+
+    @VisibleForTesting
+    boolean isKnown(@Nonnull ByteBuffer payload) {
```

Comment:
This should also be gone, I don't think it's needed anymore... Or we move the one from DefaultDataRowKeyCache here, since it's already a ByteBuffer arg